app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -47,7 +47,7 @@
         = _("Web Services")
       = miq_tab_header('ipmi', nil, {'ng-click' => "changeAuthTab('ipmi')"}) do
         = _("IPMI")
-    - elsif %w(ems_network).include?(controller_name)
+    - elsif %w(ems_network).include?(controller_name) && record.emstype == 'nuage_network'
```

Comment:
We'd like to avoid constructs like the above as much as possible and use `supports?` mechanism instead.

Reply:
I was trying to match the conditions which show the fields in the tab content, where emstype is used explicitly. In that code, embedded in an "ng-if" element is  this conditional: "#{ng_model}.emstype == 'nuage_network'" -- the idea being that the tab link should be hidden if the content is empty. If we're not using supports? to determine whether to show the content, wouldn't it be confusing to use it to use it to determine whether to show the navigation to it? If supports? is still desirable despite this, what would be the appropriate thing to check for to make sure that it only appears for nuage_network and not for other network types (to match the content condition)?

Reply:
@mzazrivec Any updates here? Given the context, that everything else here -- i.e. the fields on the tab we're trying to hide here -- is already using emstype, is it OK to leave this as-is? If not, we'd probably want to switch the ng-if conditionals to use supports? as well, but I don't know if that's possible on the javascript side, which is where those checks are happening.

Reply:
I think the point is not to add more mentions to specific providers. We are trying to eliminate them, getting to pluggable providers. So adding more would make it harder.

Could you try to model it via support?

Reply:
@Ladas I think the only reasonable way to avoid mentioning this specific provider here is also to rewrite the other place that mentions it. This code hides the tab unless emstype is nuage_network, and the other code explicitly. In particular, the code above needs to have the same checks as the code here: https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/views/layouts/angular/_multi_auth_credentials.html.haml#L234

Are there examples of using the supports bit in the angular code like this? The other challenge here is that I've never worked with the nuage_network provider, so I don't really know why it's the only network provider that needs the events tab shown (i.e. I'm not quite sure what feature/supports test would be common to nuage_network, openstack, and vmware but not to anything else), but the way the existing code is written, the content is hidden based on emstype (in the quoted line above), so I was just following the same pattern for hiding the navigation tab that linked to that content.